_conferences/android-worldwide-january-2022.md

```diff
@@ -0,0 +1,14 @@
+---
+name: "Android Worldwide"
+website: https://android-worldwide.com/
```

Comment:
The website is not mentioning the new date, is that correct?

Reply:
Correct, we'll update the website when the CFP launches. Thank you for asking :)

Reply:
When is it happening?

Reply:
That will happen on November 8th

Reply:
Ok. If you don't mind I'd hold on till that date for merging. Also because your CFP mentions that is in _trial mode_. So we can add a new event with both links valid 👍